src/CMakeLists.txt

```diff
@@ -892,6 +892,10 @@ list( APPEND SOURCES
       tracks/ui/ZoomHandle.cpp
```

Comment:
The commit command means "now", not "not", right?

Reply:
Yes, I will fix that